packages/terra-compact-interactive-list/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Updated `handleMouseDown` method for a cell.
```

Comment:
This changelog needs to be updated to reflect other updates done.

Reply:
Good catch! Updated here: https://github.com/cerner/terra-framework/pull/1950/commits/bcd1cd027dbd6870c571197300632e6f6eb42950